### DISCUSSION TITLE
Svelte support

### DISCUSSION THREAD

**Discussion by gustavopch** [konstaui/konsta#26] [General]
I don't understand why https://github.com/tailwind-mobile/tailwind-mobile/issues/2 was simply locked without any explanation. It's a very relevant issue as the README and website of this project both mention Svelte support. In case you're also locking this discussion, please at least provide some explanation, even if only to say that Svelte won't be supported, or that it's in the roadmap but work hasn't been started — literally anything instead of just locking.

**Comment by nolimits4web** [comment]
https://konstaui.com/svelte
